src/geode/geosciences/lso_input.cpp

```diff
@@ -446,6 +467,7 @@ namespace
             const auto component_id = model_.block( block_id ).component_id();
             absl::flat_hash_map< geode::index_t, geode::index_t >
                 vertex_mapping;
+            std::vector< geode::index_t > inverse_vertex_mapping;
```

Comment:
Isn't FixedArray better for small arrays ?

Reply:
True, for large array it is the same than a vector. The main change is that it says more about the purpose of the container. If you prefer vector, you should reserve the space. Then it will be identical